WalletWasabi/CoinJoin/Client/CoinJoinProcessor.cs

```diff
@@ -49,6 +49,11 @@ private async void Synchronizer_ResponseArrivedAsync(object? sender, Synchronize
 
 				var txsNotKnownByAWallet = WalletManager.FilterUnknownCoinjoins(unconfirmedCoinJoinHashes);
 
+				if (!txsNotKnownByAWallet.Any())
+				{
+					return;
```

Comment:
This is obviously good. I'm not sure it changes any behavior, otherwise things should have been not working already. Anyway, just wanted to note for those reviewers who're puzzled by this.

Reply:
Nop, this is wrong because it is unnecessary and it reveals that we don't trust in the correctness of our code (it is correct btw).

Reply:
Anyway, I removed the change.